charts/geoweb-opmet-backend/templates/opmet-deployment.yaml

```diff
@@ -149,7 +149,7 @@ spec:
         - name: publisher-volume
           mountPath: {{ .Values.opmet.publisher.DESTINATION | quote }}
       - name: {{ .Values.opmet.nginx.name }}
-        image: {{ .Values.opmet.nginx.registry }}:{{ default .Chart.AppVersion .Values.versions.opmet }}
+        image: {{ .Values.opmet.nginx.registry }}:{{ default .Chart.AppVersion .Values.opmet.nginx.version }}
```

Comment:
`default .Chart.AppVersion` should be removed because it is opmet-backend version

Reply:
Removed on latest commit